### PR TITLE
UCX: bump version to 1.2.1

### PR DESCRIPTION
**Pull request by amaslenn**

### CODE REVIEW

File: ucx.spec.in [hunk 1]
```diff
@@ -79,5 +79,7 @@ rm -f %{buildroot}%{_libdir}/*.la
 %postun -p /sbin/ldconfig
 
 %changelog
+* Mon Aug 21 2017 Andrey Maslennikov <andreyma@mellanox.com> 1.2.1-1
+- Spec file now compy with Fedora guidelines
```
Comment (alinask) [inline ucx.spec.in]: comply

Comment (amaslenn) [inline ucx.spec.in]: Fixed, thanks.

**Comment by amaslenn** [comment]
@shamisp @yosefe please review

**Comment by amaslenn** [comment]
bot:bgate:retest

**Comment by amaslenn** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4346/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4345/ for details (Mellanox internal link).

**Comment by amaslenn** [comment]
bot:bgate:retest

File: ucx.spec.in [hunk 2]
```diff
@@ -79,5 +79,7 @@ rm -f %{buildroot}%{_libdir}/*.la
 %postun -p /sbin/ldconfig
 
 %changelog
+* Mon Aug 21 2017 Andrey Maslennikov <andreyma@mellanox.com> 1.2.1-1
+- Spec file now compies with Fedora guidelines
```
Comment (miked-mellanox) [inline ucx.spec.in]: s/compies/complies

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4349/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4355/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4356/ for details (Mellanox internal link).

**Pull request merged**